### DISCUSSION TITLE
Online search system

### DISCUSSION THREAD

**Discussion by gogo25171** [osmandapp/OsmAnd#13302] [Q&A] [answered]
Hello,
I would like you to put the possibility of making an online system to search for points like a bit for route calculation.
Thank you in advance, truthfully yours.

**Comment by Lee-Carre** [comment, marked as answer]
This is already implemented. I've used it myself.

**Reply by gogo25171** [reply]
Hi, you can tell me how you do it because the main problem is the street search which is not efficient and in addition which is extremely long. Thank you in advance, truthfully yours.

**Reply by Lee-Carre** [reply]
Hmm, I was about to type a description from memory, but (having not used it in a while) decided to check first.

Good thing I did, because it seems that the function may have been removed in v4. Your question makes a whole lot more sense, now 😄.

In v3 (this is definitely from memory), it went something like this;
1. Open the search UI
2. Enter your keywords
3. Especially if no results (or only a few) are matched, and you told it to increase the search radius several times, it would prompt to perform an online search (I'm guessing that was a Nominatim query, or similar)
4. Else, scrolling down to the bottom would show an option to trigger on online search anyway, even with many offline matches

Given its absence in v4, instead you might try searching on OSM.org and then exporting the result to OsmAnd. Else, better, would be to install Acastus Photon (available via F-Droid) which is a dedicated app for such queries, and makes transfer of results into other apps very easy.

**Reply by gogo25171** [reply]
OK, thanks

**Reply by Lee-Carre** [reply]
If that satisfies your original query, please mark this as the answer (and possibly up-vote) for the benefit of others.

**Reply by Lee-Carre** [reply]
@gogo25171

Seems that I was incorrect;
1. Search
2. Categories tab
3. Scroll down to the bottom
4. Select the entry for Online Search

**Reply by gogo25171** [reply]
OK, thanks

**Reply by Lee-Carre** [reply]
@gogo25171 would you mark this response as the answer (& possibly up-vote) for the benefit of others.
